R/FilterStateChoices.R

```diff
@@ -578,6 +582,31 @@ ChoicesFilterState <- R6::R6Class( # nolint
       )
     },
 
+    server_inputs_fixed = function(id) {
+      moduleServer(
+        id = id,
+        function(input, output, session) {
+          logger::log_trace("ChoicesFilterState$server initializing, dataname: { private$dataname }")
+
+          output$selection <- renderUI({
+            countsnow <- unname(table(factor(private$x_reactive(), levels = private$choices)))
+            countsmax <- private$choices_counts
+
+            ind <- private$choices %in% private$selected()
+            countBars(
+              inputId = session$ns("labels"),
+              choices = private$selected(),
```

Comment:
Just to secure this renderUI from being retriggered. (Even though selected is protected from being updated when fixed)
```suggestion
            ind <- private$choices %in% shiny::isolate(private$selected())
            countBars(
              inputId = session$ns("labels"),
              choices = shiny::isolate(private$selected()),
```